guides/17_Migration_to_3_0.md

```diff
@@ -60,22 +71,98 @@ change it to
 await messagingSession.start();
 ```
 
+### `MeetingSessionPOSTLogger` to `POSTLogger`
+
+With Amazon Chime SDK for JavaScript v3, we have removed `MeetingSessionConfiguration` dependency from the `MeetingSessionPOSTLogger`. The `MeetingSessionPOSTLogger` is now renamed to `POSTLogger`.
+
+Builders can add optional fields to `POSTLogger` which will be included when `POSTLogger` sends the HTTP POST logs to builder provided URL:
+
+1. HTTP POST request `headers`.
+2. Informational `metadata`. For example, `meetingId`, `attendeeId`, `appName`. This `metadata` depends on builders and `POSTLogger` will include this `metadata` as part of the request body when making the HTTP POST requests to builder provided URL.
+
+#### Constructor
+
+Before in 2.x:
+
+```typescript
+// You need responses from server-side Chime API. See below for details.
+const meetingResponse = // The response from the CreateMeeting API action.
+const attendeeResponse = // The response from the CreateAttendee API action.
+const meetingSessionConfiguration = new MeetingSessionConfiguration(meetingResponse, attendeeResponse);
+
+const meetingSessionPOSTLogger = new MeetingSessionPOSTLogger(
+  'SDK',
+  configuration,
+  20, // LOGGER_BATCH_SIZE
+  2000, // LOGGER_INTERVAL_MS
+  URL,
+  LogLevel.INFO
+);
+```
+
+After in 3.x:
+
+```typescript
+// You need responses from server-side Chime API. See below for details.
+const meetingResponse = // The response from the CreateMeeting API action.
+const attendeeResponse = // The response from the CreateAttendee API action.
+
+const postLoggerOptions = {
+  metadata: {
+    appName: 'SDK',
+    meetingId: meetingResponse.Meeting.MeetingId,
+    attendeeId: attendeeResponse.Attendee.AttendeeId,
+  },
+};
+const logger = new POSTLogger(
+  20, // LOGGER_BATCH_SIZE
+  2000, // LOGGER_INTERVAL_MS
+  URL,
+  LogLevel.INFO,
+  postLoggerOptions
+);
+```
+
+#### Set `metadata` after `POSTLogger` creation
+
+```typescript
+const logger = new POSTLogger(
+  20, // LOGGER_BATCH_SIZE
+  2000, // LOGGER_INTERVAL_MS
+  URL,
+  LogLevel.INFO
+);
+
+// You need responses from server-side Chime API. See below for details.
+const meetingResponse = // The response from the CreateMeeting API action.
+const attendeeResponse = // The response from the CreateAttendee API action.
+
+const metadata = {
```

Comment:
Seem like this could be in the same example as above.

Reply:
It just shows adding metadata after the object creation. First example, shows it as part of the constructor.

Reply:
Then could we add one sentence explain this use case?

Reply:
I added a brief explanation when that can be a use case, good point!